### ISSUE TITLE
Ensure correct `graphicId` HTML element / labels

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#4473]
See https://github.com/mozilla/fxa/pull/4465#discussion_r389880425

Places in which we set a graphic through a class on a `div` with a `background-image` need some `a11y` love.

These should preferably be `img` tags for screen readers, however, our image file names receive a hash via grunt on stage/prod build so a simple path to the icon will break in stage. A `data:image/svg+xml;utf8,<svg...` src could be the way to go, however, this may be tricky since we use `getGraphicsId` to check for which class name to set on the div.

At the very least, we can instead place `role="img"` and `aria-label="alt text"` on the divs which will read as images. Reference the link above to see how this was done in one file. Closing this ticket should explore if we can use `img` but if not, proper roles/aria-labels on all divs that receive a `graphicId` will suffice.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-1328)
┆Issue Number: FXA-1328

**Comment by imskr** [comment]
I would like to fix this! @LZoog

**Comment by jodyheavener** [comment]
@imskr You're welcome to take this issue. Thank you!

**Comment by imskr** [comment]
@jodyheavener Do I have to make changes in `*.mustache` file located [here](https://github.com/mozilla/fxa/tree/master/packages/fxa-content-server/app/scripts/templates)

**Comment by jodyheavener** [comment]
@imskr Correct, those are the template files we're looking to audit.

**Comment by imskr** [comment]
@jodyheavener What should I write in `aria-label="alt text"` in place of alt text in each file or it should be `aria-label="alt text"` only?

**Comment by jodyheavener** [comment]
@imskr Great question. I would recommend setting the aria-label for each occurrence with what is being presented to the user. If you check out https://github.com/mozilla/fxa/pull/4465#discussion_r389880425 you'll see the  label is set to `{{#t}}Connect another device{{/t}}` because we're presenting a screen to connect additional devices. You can use other copy on the page, or the file name of the image, to infer what the label should be.

Also note the use of `{{#t}}string{{/t}}` as this is how we are setting up strings for translation.
